File: db4on/core/src/core/com/db4o/TransactionParticipant.cs
namespace com.db4o
{
	/// <exclude></exclude>
	public interface TransactionParticipant
	{
		void Commit(com.db4o.Transaction transaction);

		void Rollback(com.db4o.Transaction transaction);

		void Dispose(com.db4o.Transaction transaction);
	}
}

File: db4oj/core/src/com/db4o/internal/TransactionParticipant.java
/* Copyright (C) 2007   db4objects Inc.   http://www.db4o.com */
package com.db4o.internal;

/**
 * @exclude
 */
public interface TransactionParticipant {	

	void commit(Transaction transaction);

	void rollback(Transaction transaction);
	
	void dispose(Transaction transaction);
}
